Source/WebKit/UIProcess/WebsiteData/WebsiteDataStore.h

```diff
@@ -543,12 +528,16 @@ class WebsiteDataStore : public API::ObjectImpl<API::Object::Type::WebsiteDataSt
 #endif
 
     String migrateMediaKeysStorageIfNecessary(const String& directory);
+    void resolveDirectories(WebsiteDataStoreConfiguration::Directories&);
```

Comment:
Can this function be marked as `static`? I took a quick look at the implementation and it seems like maybe?
If so, could it at least be marked as `const`?